CHANGELOG.md

```diff
@@ -6,6 +6,131 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+- don't throw unnecessary exception on peer connect by @almogdepaz in https://github.com/Chia-Network/chia-blockchain/pull/14108
+- Return existing CAT wallet instead of raising by @Quexington in https://github.com/Chia-Network/chia-blockchain/pull/14101
+- server: Resolve peers in harvester and timelord startup also by @xdustinface in https://github.com/Chia-Network/chia-blockchain/pull/14161
+- bump default bladebit version to 2.0.1 in install-plotter.sh by @arvidn in https://github.com/Chia-Network/chia-blockchain/pull/14181
+- Add MempoolRemoveReason, which represents why we are removing a MempoolItem by @aqk in https://github.com/Chia-Network/chia-blockchain/pull/14263
+- wallet: Drop duplicate puzzle hashes assert by @xdustinface in https://github.com/Chia-Network/chia-blockchain/pull/14287
+- Check if a SpendBundle was included in a block BEFORE removing it from the Mempool by @aqk in https://github.com/Chia-Network/chia-blockchain/pull/14293
+- Fix fee and unit amounts when coin splitting or viewing coins by @jack60612 in https://github.com/Chia-Network/chia-blockchain/pull/14172
+- disallow empty SpendBundles in the mempool by @arvidn in https://github.com/Chia-Network/chia-blockchain/pull/14305
+- Check for valid timeout and log error  by @emlowe in https://github.com/Chia-Network/chia-blockchain/pull/14304
+- add rate limits numbers to none response type by @almogdepaz in https://github.com/Chia-Network/chia-blockchain/pull/14184
+- Fixed condition to check for wallet_id by @steppsr in https://github.com/Chia-Network/chia-blockchain/pull/14306
+- fix offer cancel balance by @trepca in https://github.com/Chia-Network/chia-blockchain/pull/14342
+- Fix bulk tx by @ytx1991 in https://github.com/Chia-Network/chia-blockchain/pull/14372
+- Make the `--id` flag on cancel_offer required by @Quexington in https://github.com/Chia-Network/chia-blockchain/pull/14376
+- Fix target puz hash index for bulk mint nfts with airdrop by @geoffwalmsley in https://github.com/Chia-Network/chia-blockchain/pull/14370
+- Fix chia wallet coins cli bugs by @jack60612 in https://github.com/Chia-Network/chia-blockchain/pull/14388
+- Fixed wallet state if db is rolled back by @wjblanke in https://github.com/Chia-Network/chia-blockchain/pull/14384
+- Add some extra checks for size and existence to notification backend by @Quexington in https://github.com/Chia-Network/chia-blockchain/pull/14398
+- fix sending of the `new_on_chain_notification` state change message by @paninaro in https://github.com/Chia-Network/chia-blockchain/pull/14410
+- Add a persitent id table to notification store by @Quexington in https://github.com/Chia-Network/chia-blockchain/pull/14418
+- Check incoming notification hints by @Quexington in https://github.com/Chia-Network/chia-blockchain/pull/14450
+- Quex.offer mod tests by @Quexington in https://github.com/Chia-Network/chia-blockchain/pull/14478
```

Comment:
This is either internal or needs explanation.

```suggestion
```

Reply:
I guess this would be @Quexington as well.

Reply:
Removed this line and added a new one about offer updates in general